docs/release-notes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Release Notes
+---
+
+## v2.0.0
+
+### New Features
+
+- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation)
+LNS Protocol (v2.0.5)
+- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation)
+CUPS Protocol (v2.0.5) for credential management and firmware upgrade
+- LoRaWAN Network Server and Basics™ Station can be decoupled, allowing the
+possibility to connect multiple concentrators on a single LoRaWAN Network Server
+instance
+- Support for secure communication between LoRaWAN Network Server and Basics™
+Station
+- Support for running LoRaWAN Network Server on [Azure IoT Edge for Linux on
+Windows](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-for-linux-on-windows?view=iotedge-2018-06)
+- Regional support for AS923 and CN470 (Regional Parameters v1.0.3rA and
+RP002-1.0.3 revisions)
+- Support for running LNS on a separate Edge or Cloud device
+- First class support for Class C devices
+- Observability:
+  - Integration with Azure Monitor
+  - Exposing a Prometheus endpoint for metrics scraping
+  - default monitoring for edgeAgent and edgeHub modules
+  - distributed tracing
```

Comment:
We should be more detailed on what this means

Reply:
I think we should remove it, we don't really have support for distributed tracing.